### PR TITLE
quiet compiler

### PR DESCRIPTION
**Pull request by sethalves**
- quiet compiler

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 49ee25123860991baad68cf7ed694a76eefcf5ea available at: https://gist.github.com/00ff67bfb3dfae11dece<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5790.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5790.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 93dc0714f7944d6be42cbc7d579541056c4b6752 available at: https://gist.github.com/00ff67bfb3dfae11dece<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5790.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5790.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 61c9da1d2719ee991c5c864ff50f4af04a0f5d88 available at: https://gist.github.com/00ff67bfb3dfae11dece<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5790.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5790.dmg

File: interface/src/FileLogger.cpp [hunk 1]
```diff
@@ -53,7 +53,7 @@ class FilePersistThread : public GenericQueueThread < QString > {
 protected:
     void rollFileIfNecessary(QFile& file) {
         uint64_t now = usecTimestampNow();
-        if ((file.size() > MAX_LOG_SIZE) || (now - _lastRollTime) > MAX_LOG_AGE_USECS) {
+        if ((file.size() > (qint64)MAX_LOG_SIZE) || (now - _lastRollTime) > MAX_LOG_AGE_USECS) {
```
Comment (ZappoMan) [inline interface/src/FileLogger.cpp]: why not just change the original type for MAX_LOG_SIZE?

File: libraries/entities/src/EntityScriptingInterface.cpp [hunk 2]
```diff
@@ -644,7 +644,7 @@ QVector<QUuid> EntityScriptingInterface::getActionIDs(const QUuid& entityID) {
     actionWorker(entityID, [&](EntitySimulation* simulation, EntityItemPointer entity) {
             QList<QUuid> actionIDs = entity->getActionIDs();
             result = QVector<QUuid>::fromList(actionIDs);
-            return true;
+            return false; // don't send an edit packet
```
Comment (ZappoMan) [inline libraries/entities/src/EntityScriptingInterface.cpp]: this is clearly more than a warning fix. Can you clarify the change here. Was this a bug fix? how do we verify the new  behavior

File: libraries/ui/src/VrMenu.cpp [hunk 3]
```diff
@@ -167,7 +173,10 @@ void VrMenu::insertAction(QAction* before, QAction* action) {
     }
     QObject* menu = beforeQml->parent();
     QVariant returnedValue;
-    bool invokeResult = QMetaObject::invokeMethod(this, "insertItem", Qt::DirectConnection,
+    #ifndef QT_NO_DEBUG
```
Comment (ZappoMan) [inline libraries/ui/src/VrMenu.cpp]: I wonder if we should make ourselves a macro for this construct... something like...

```
ASSERT_VARIABLE(bool invokeResult =) QMetaObject::invokeMethod(this, "insertItem", Qt::DirectConnection,
         Q_RETURN_ARG(QVariant, returnedValue),
         Q_ARG(QVariant, QVariant::fromValue(menu)),
         Q_ARG(QVariant, QVariant::fromValue(beforeQml)),
         Q_ARG(QVariant, QVariant::fromValue(action->text())));
```

File: libraries/entities/src/EntityScriptingInterface.cpp (hunk 2 above)
Comment (sethalves) [inline libraries/entities/src/EntityScriptingInterface.cpp]: You're right -- it was just something I noticed and fixed.  We were sending "update" packets during the read operations.  I don't expect any noticeable change.  Test by making sure the grab scripts still work.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 414d9162efb6bc362f93013dbf9b771f249cef61 available at: https://gist.github.com/00ff67bfb3dfae11dece<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5790.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5790.dmg

File: libraries/ui/src/VrMenu.cpp (hunk 3 above)
Comment (sethalves) [inline libraries/ui/src/VrMenu.cpp]: this would litter the code with smiley-faces!

Comment (ZappoMan) [inline libraries/ui/src/VrMenu.cpp]: lolz - ok - we wouldn't want that!

**Pull request merged**